packages/destination-actions/src/destinations/gainsight-px-cloud-action/index.ts

```diff
@@ -96,13 +65,36 @@ const destination: DestinationDefinition<Settings> = {
       password: ''
     }
   },
-  presets,
   actions: {
-    identifyUser,
-    groupIdentify,
-    trackEvent,
-    trackPageView
-  }
+    // Only one action, event data is simply copied from Segment payload and sent to Gainsight PX
+    sendEvent
+  },
+  presets: [
+    {
+      name: 'Identify User',
+      subscribe: 'type = "identify"',
+      partnerAction: 'sendEvent',
+      mapping: defaultValues(sendEvent.fields)
```

Comment:
Move `defaultValues` as a constant at the top of the file: 
`const defaultVals = { ...defaultValues(sendEvent.fields) }`
```suggestion
      mapping: defaultVals
```